packages/appsync-modelgen-plugin/scripts/generateStandaloneValidationFunction.ts

```diff
@@ -0,0 +1,14 @@
+import Ajv from 'ajv';
+import modelIntrospectionSchemaDefinition from '../schemas/introspection/1/ModelIntrospectionSchema.json'
```

Comment:
nit for future: Since we're versioning the MIS definition and this script generates the standalone for version 1 of MIS, we can maybe make it more generic so that it accepts the version input and outputs one standalone per version.